.travis.yml

```diff
@@ -26,4 +26,3 @@ script:
   - export PYTHONPATH=$PYTHONPATH:$PWD
   - flake8 .
   - py.test -vs tests/
-  - python benchmark.py
```

Comment:
We should either still run the benchmark explicitly or convert some it to pytest conventions and run `py.test -vs examples`.